src/promptflow-evals/promptflow/evals/evaluators/chat/__init__.py

```diff
@@ -0,0 +1,189 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
+
+__path__ = __import__("pkgutil").extend_path(__path__, __name__)  # type: ignore
+
+from promptflow.entities import AzureOpenAIConnection
+from promptflow.evals.evaluators import GroundednessEvaluator, RelevanceEvaluator, CoherenceEvaluator, FluencyEvaluator
+from typing import List, Dict
+from concurrent.futures import ThreadPoolExecutor, as_completed
+import json
+import logging
+import numpy as np
+
+
+logger = logging.getLogger(__name__)
+
+
+class ChatEvaluator:
+    def __init__(self, model_config: AzureOpenAIConnection, deployment_name: str):
```

Comment:
Can we have a param which allows to just do evaluation for last turn ?

Reply:
Sure thing. Just curious is that a requirement from customer?

Assume this optional param needs to be added to the `__callable__` not `__init__`, correct?

Reply:
To __init__ since that will be configuration that customer can pass. For parallel can be have it as configuration param ?

Reply:
This will be needed by evaluate API when target is present.